projects/storefrontapp-e2e-cypress/cypress/helpers/vendor/epd-visualization/visual-picking-tab.ts

```diff
@@ -0,0 +1,150 @@
+export function configureDefaultProduct() {
+  cy.window().then((win) => win.sessionStorage.clear());
+
+  // We need to ensure we use control the screen width, since the animation controls are removed at lower breakpoints.
+  cy.viewport(1200, 800); // Bootstrap 4 XL breakpoint
+
+  cy.intercept(
+    'GET',
+    `${Cypress.env('OCC_PREFIX')}/${Cypress.env(
+      'BASE_SITE'
+    )}/cms/pages?pageType=ProductPage**`
+  ).as('productPage');
+
+  cy.visit('/product/CX704/7%E2%80%9Dx12%E2%80%9D-mini-metal-lathe');
+  cy.wait(`@productPage`);
+
+  cy.get('cx-breadcrumb').should('contain', 'Home');
+  cy.get('cx-breadcrumb').should('contain', 'Lathes');
+  cy.get('cx-breadcrumb').should('contain', 'Craftex');
+
+  cy.get('cx-epd-visualization-visual-picking-tab');
+}
+
+export function verifyTabbingOrder() {
+  // Ensure the spare parts tab is active
+  cy.get('cx-tab-paragraph-container > button').contains('Spare Parts').click();
+
+  cy.get('cx-epd-visualization-viewer', { timeout: 50000 }).should(
```

Comment:
Why such a big timeout?

Reply:
This was sometimes timing out at 30s for me when using the CX704 model. The viewer is hidden until the scene structure has loaded and geometry and materials have been downloaded (at which point the spinner is hidden). I have a 1GB/s internet connection, but when using an SAP VPN connection to connect to machines in Europe I sometimes experience very high latency (sometimes higher than 500ms) and very low performance when accessing servers in Europe (probably due to strange routing).